docs/csharp/language-reference/keywords/value-types.md

```diff
@@ -37,16 +39,18 @@ The value types consist of two main categories:
   
  Unlike reference types, a value type cannot contain the `null` value. However, the [nullable types](../../../csharp/programming-guide/nullable-types/index.md) feature does allow for value types to be assigned to `null`.  
   
- Each value type has an implicit default constructor that initializes the default value of that type. For information about default values of value types, see [Default Values Table](../../../csharp/language-reference/keywords/default-values-table.md).  
+ Each value type has an implicit default constructor that initializes the default value of that type. For information about default values of value types, see [Default values table](default-values-table.md).  
   
-## Main Features of Simple Types  
- All of the simple types -- those integral to the C# language -- are aliases of the .NET Framework System types. For example, [int](../../../csharp/language-reference/keywords/int.md) is an alias of <xref:System.Int32?displayProperty=nameWithType>. For a complete list of aliases, see [Built-In Types Table](../../../csharp/language-reference/keywords/built-in-types-table.md).  
+## Main features of simple types
+
+ All of the simple types -- those integral to the C# language -- are aliases of the .NET <xref:System> types. For example, [int](int.md) is an alias of <xref:System.Int32?displayProperty=nameWithType>. For a complete list of aliases, see [Built-in types table](built-in-types-table.md).  
```

Comment:
Here we have a different category "simple types". The categorization and this section should match.

We shouldn't use the word "integral". It is too close to integer, and there are better synonyms available.

I would remove the article in "are aliases of the .NET System types"

Reply:
Please check new updates; I've updated this section a lot.